app/ai/voice/llm/__init__.py

```
@@ -0,0 +1,29 @@
+    "LLMSdk",
+    "LLMConfiguration",
+    # Azure
+    "AzureConfig",
+    "build_azure_llm",
+    # Google Vertex (Gemini)
+    "VertexConfig",
+    "build_vertex_llm",
+    # Claude on Vertex
+    "ClaudeVertexConfig",
+    "build_claude_vertex_llm",
+]
```

Ruff (0.15.7)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)